test/mocha/screw-ie8.js

```diff
@@ -0,0 +1,26 @@
+var assert = require("assert");
+var uglify = require("../../");
+
+describe("screw-ie8", function () {
+    it("Should be able to minify() with undefined as catch parameter in a try...catch statement", function () {
+        assert.strictEqual(
+            uglify.minify(
+                "function a(b){\
+                    try {\
+                        throw 'Stuff';\
+                    } catch (undefined) {\
+                        console.log('caught: ' + undefined);\
+                    }\
+                    console.log('undefined is ' + undefined);\
+                    return b === undefined;\
+                };", {
+                    fromString: true,
+                    compress: { screw_ie8: false },
+                    mangle: { screw_ie8: false },
+                    output: { screw_ie8: false }
```

Comment:
Please remove compress, mangle and output sections. The intent was to verify that minify with default options handles `catch` scopes correctly. It also verifies that `screw_ie8: true` is the default.